approaches/bulk-data-loading/src/InventoryVerifier.js

```diff
@@ -0,0 +1,89 @@
+import {GqlMutations} from "./GqlMutations.js";
+import {VendiaClient} from "./VendiaClient.js";
+import {checkDelay, inventoryRecordSize, maxInventoryChecks} from "./Constants.js";
+
+export class InventoryVerifier {
```

Comment:
We might consider a Graphql subscription for this? 

Would be something like
`vendiaClient.entities.inventory.onAdd(() => totalInventory++)

Reply:
I like the idea but was afraid this would be a step too far for those who need help with mutation batching.  I would like to add a feature example to highlight the value / utility of GQL subscriptions (but will leave that for another day).